pkg/platform/microservice/configFiles/repo.go

```diff
@@ -104,10 +113,10 @@ func (r k8sRepo) AddEntryToConfigFiles(applicationID string, environment string,
 func (r k8sRepo) RemoveEntryFromConfigFiles(applicationID string, environment string, microserviceID string, key string) error {
 
 	logContext := r.logContext.WithFields(logrus.Fields{
-		"method": "GetConfigFilesNamesList",
-		"application_id": applicationID,
+		"method":          "GetConfigFilesNamesList",
```

Comment:
```suggestion
		"method":          "RemoveEntryFromConfigFiles",
```

This should be the name of the method where this logger is defined.